apps/examples/testcase/le_tc/kernel/tc_libc_stdlib.c

```diff
@@ -46,24 +46,35 @@
 #define BSEARCH_ARRSIZE 10
 
 /**
-* @fn                   :compare
-* @description          :Function for tc_libc_stdlib_qsort
-* @return               :int
-*/
+ *@fn					:thread_func
+ *@description			:Function for tc_libc_stdlib_abort
+ *@return				:void*
+ */
+static void *thread_func(void *arg)
+{
+	abort();
```

Comment:
Pradeep left a comment for this at previous P.R. What is an answer?
pradeepns85 10 days ago Member
Isn't this test case stop execution after calling abort ? How to continue the test case after this test case